core/domain/email_manager.py

```diff
@@ -77,6 +77,34 @@ def log_new_error(*args, **kwargs):
         'subject': _PLACEHOLDER_SUBJECT,
         'html_body': _PLACEHOLDER_HTML_BODY,
     })
+
+EXPLORATION_ROLE_MANAGER = 'Manager rights'
```

Comment:
In the final email, this occurs in the middle of a sentence, and it seems strange to capitalize the words mid-stream Like This. However, this is not obvious at all until you look at the single golden email string.